ibis/expr/tests/test_value_exprs.py

```diff
@@ -1322,6 +1322,17 @@ def test_large_timestamp():
     assert result == expected
 
 
+@pytest.mark.parametrize('tz', [
+    None,
+    'UTC'
+])
+def test_timestamp_with_timezone(tz):
```

Comment:
Can you add a second that checks that the `timezone` is propagated to the `type()` of the resulting expression?